src/config.py

```
@@ -0,0 +1,111 @@
+"""
+Agent-News: Crypto News and Sentiment Analysis System
+Core configuration and settings management
+"""
+
+import os
+from typing import List, Optional
+from pydantic import BaseSettings, Field
```

Pylint (3.3.7)

[error] Unable to import 'pydantic'

(E0401)